app/pods/application/route.js

```diff
@@ -1,30 +1,50 @@
+import { action } from '@ember/object';
 import Route from '@ember/routing/route';
-import ENV from 'frontend-kaleidos/config/environment';
-import ApplicationRouteMixin from 'ember-simple-auth/mixins/application-route-mixin';
 import { inject as service } from '@ember/service';
 
-export default Route.extend(ApplicationRouteMixin, {
-  moment: service(),
-  intl: service(),
-  currentSession: service(),
-  fileService: service(),
-  routeAfterAuthentication: 'agendas',
-  router: service(),
-  metrics: service(),
+function isSupportedBrowser() {
```

Comment:
We gebruiken [elders](https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/45b396bc483ce05cc3d45fe5d0cd0ec35a0a68e4/app/components/web-components/vl-rdfa-editor.js#L17) als https://github.com/willviles/ember-useragent ... misschien de moeite om mee te nemen?